msm/src/lookups.rs

```diff
@@ -0,0 +1,112 @@
+//! Instantiate the MVLookup protocol for the MSM project.
```

Comment:
Note: Let's move it later. It will be done in https://github.com/o1-labs/proof-systems/pull/1891